src/langsmith/self-host-sso.mdx

```diff
@@ -13,7 +13,13 @@ Our implementation supports almost anything that is OIDC compliant, with a few e
 
 By default, LangSmith Self-Hosted supports the `Authorization Code` flow with `Client Secret`. In this version of the flow, your client secret is stored security in the LangSmith platform (not on the frontend) and used for authentication and establishing auth sessions.
 
-### Requirements
+### Prerequisites
+
+* You must be self-hosted and on an Enterprise plan.
+* Your IdP must support the `Authorization Code` flow with `Client Secret`.
+* Your IdP must support using an external discovery/issuer URL. We will use this to fetch the necessary routes and keys for your IdP.
+* You must provide the `OIDC`, `email`, and `profile` scopes to LangSmith. We use these to fetch the necessary user information and email for your users.
+* *Note*: Langsmith SSO is only supported over `https`.
 
 <Note>
```

Comment:
Seems like this doesn't need a note callout and should be higher in the doc, like in the overview section